File: Product.Domain/Model/ProductDetail.cs
using System;


namespace Product.Domain
{
    public class ProductDetail
    {
        public Guid? Id { get; set; }
        public string ProductName { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
        public virtual PriceDetail PriceDetail { get; set; }
    }
}

File: Web/Product/src/app/product-overview/model/product-detail.ts
export interface IProductDetail {
    id:any;
    productName:string;
    description:string;
    isActive:boolean;
    priceDetail:IPriceDetail;
}

export interface IPriceDetail{
    id:any;
    price:number;
    productId:any;
    isActive:boolean;
}
